amazonclouddrive/amazonclouddrive.go

```diff
@@ -862,6 +966,50 @@ func (o *Object) Remove() error {
 	return err
 }
 
+// Restore an object
+func (o *Object) Restore() error {
```

Comment:
This should have a small initial letter `restore` so it isn't part of the public API of this package.